Assets/Scripts/Models/Furniture.cs

```diff
@@ -143,15 +146,25 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
+    /// <summary>
+    /// This event will trigger when the furniture has been changed.
```

Comment:
That will not work since then you will think that it is for a particular property and not for any. I think that changed is fine as a name, since a new name would have to enclose many possible changed events. Like sometimes changed can be called because a neighbour changed, which would make the sprite change (but no property of the furniture changed). You can also trigger it from Lua, without any change. The description could tell a bit more. For now it is mostly used to update the sprite.